authors/apps/highlights/exceptions.py

```diff
@@ -0,0 +1,41 @@
+from rest_framework.exceptions import APIException
+
+
+class FieldNotSupported(APIException):
+    """
+    Exception for an unsupported field
+    """
+    status_code = 404
+    default_detail = "Field must be; title, body or description"
+
+
+class StartIndexError(APIException):
+    """
+    Exception for Start index greater than End index
+    """
+    status_code = 400
+    default_detail = "Start index cannot be greater than End index"
+
+
+class IndexOutOfRange(APIException):
+    """
+    Exception for Index beyond String length
+    """
+    status_code = 400
+    default_detail = "end_index cannot be greater than field length"
+
+
+class NotPermittedError(APIException):
+    """
+    Exception for no permission to perform an action
+    """
+    status_code = 400
+    default_detail = "You cannot delete or update hightlight that is noy yours"
```

Comment:
Hi @matthenge ,
Please fix the spelling mistake in the exception `You cannot delete or update hightlight that is noy yours`
to You cannot delete or update **highlight** that is **not** yours.

Reply:
Already fixed. Thanks for catching that